### ISSUE TITLE
Information banner is invisible.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [hardinfo2/hardinfo2#114]
#### Errata

![`@{'Type'='Screenshot'; 'Date'='+20241126T111846+0000[GMT][ca-iso8601]'}#.PNG`](https://github.com/user-attachments/assets/b22974f7-3c78-46c6-abf2-590509c1b814)

#### Environment

I'm using [`fedora.pkgs.org/41/fedora-updates-x86_64/hardinfo2-2.2.1-1.fc41.x86_64.rpm`](https://fedora.pkgs.org/41/fedora-updates-x86_64/hardinfo2-2.2.1-1.fc41.x86_64.rpm.html) on:

1. ```bash
   #!/usr/bin/env -S bash
   kinfo
   ```

1. ```YAML
   #!/usr/bin/env -S xdg-open
   Operating System: Fedora Linux 41
   KDE Plasma Version: 6.2.3
   KDE Frameworks Version: 6.8.0
   Qt Version: 6.8.0
   Kernel Version: 6.11.10-300.fc41.x86_64 (64-bit)
   Graphics Platform: Wayland
   Processors: 12 × AMD Ryzen 5 7600X 6-Core Processor
   Memory: 30.4 GiB of RAM
   Graphics Processor: AMD Radeon RX 5700
   ```

I'll share a `hardinfo2`-generated report, if of use.

**Comment by hwspeedy** [comment]
@RokeJulianLockhart 
CC: @topazus 

Thanx, 
The hardinfo2 - missing "group add" information error is fixed in release 2.2.4.
We have had a lot of releases - but @topazus will probably update soon.

We have 2 tasks for Redhat/Fedora branch: 
1. update to 2.2.4 (Bug Fix)
2. get sysbench into EL10 (#113), thanx.

Read about bug fix release 2.2.4 here:
https://github.com/hardinfo2/hardinfo2/releases/tag/release-2.2.4

If you want - you can try the CPack build until the Redhat/Fedora repo is updated. 
https://github.com/hardinfo2/hardinfo2/releases/download/release-2.2.3pre/hardinfo2-2.2.3-FedoraLinux-41.x86_64.rpm

**Comment by hwspeedy** [comment]
@topazus 

Hardinfo2 - EPEL8 is missing below patch from git-master to build (Fails the metainfo file check included in fedora/redhat spec file).

https://github.com/hardinfo2/hardinfo2/commit/75a4f993d91242e6202e252acd8cd3773238279b

**Comment by RokeJulianLockhart** [comment]
> The hardinfo2 - missing "group add" information error is fixed in release 2.2.4.

@hwspeedy, I'll close as completed, then?
